python/lsst/pipe/tasks/diff_matched_tract_catalog.py

```diff
@@ -301,5 +537,221 @@ def run(
             cat_matched.index = cat_left.index
         cat_matched.columns.values[len(cat_target.columns):] = [f'refcat_{col}' for col in cat_right.columns]
 
-        retStruct = pipeBase.Struct(cat_matched=cat_matched)
+        # Add/compute distance columns
+        coord1_target_err, coord2_target_err = config.columns_target_coord_err
+        column_dist, column_dist_err = 'dr', 'drErr'
```

Comment:
Is this the name that the columns end up with in the table? If so could it be something more descriptive?

Reply:
It was meant to be consistent with the default (pixel) coordinate naming of x and y, but I suppose `coord_x`,`coord_y` and `distance` would be clearer.